x-pack/legacy/plugins/ml/public/overview/components/analytics_panel/analytics_panel.tsx

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, Fragment, useState, useEffect } from 'react';
+import {
+  EuiButton,
+  EuiButtonEmpty,
+  EuiCallOut,
+  EuiEmptyPrompt,
+  EuiLoadingSpinner,
+  EuiPanel,
+  EuiSpacer,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { AnalyticsTable } from './table';
+import { getAnalyticsFactory } from '../../../data_frame_analytics/pages/analytics_management/services/analytics_service';
+import { DataFrameAnalyticsListRow } from '../../../data_frame_analytics/pages/analytics_management/components/analytics_list/common';
+// TODO: panels can be smaller when empty
+export const AnalyticsPanel: FC = () => {
+  const [analytics, setAnalytics] = useState<DataFrameAnalyticsListRow[]>([]);
+  const [errorMessage, setErrorMessage] = useState<any>(undefined);
+  const [isInitialized, setIsInitialized] = useState(false);
+
+  const getAnalytics = getAnalyticsFactory(setAnalytics, setErrorMessage, setIsInitialized, false);
+
+  useEffect(() => {
+    getAnalytics(true);
+  }, []);
+
+  const onRefresh = () => {
+    getAnalytics(true);
+  };
+
+  const errorDisplay = (
+    <Fragment>
+      <EuiCallOut
+        title={i18n.translate('xpack.ml.overview.analyticsList.errorPromptTitle', {
+          defaultMessage: 'An error occurred getting the data frame analytics list.',
+        })}
+        color="danger"
+        iconType="alert"
+      >
+        <pre>{JSON.stringify(errorMessage)}</pre>
+      </EuiCallOut>
+    </Fragment>
+  );
+
+  return (
+    <EuiPanel className="mlOverviewPanel">
+      {typeof errorMessage !== 'undefined' && errorDisplay}
+      {isInitialized === false && <EuiLoadingSpinner />}     
+      {isInitialized === true && analytics.length === 0 && (
+        <EuiEmptyPrompt
+          iconType="createAdvancedJob"
+          title={<h2>Create your first analytics job</h2>}
+          body={
+            <Fragment>
+              <p>
+                Data frame analytics enable you to perform different analyses of your data and
+                annotate it with the results. As part of its output, data frame analytics appends
```

Comment:
I think the second sentence should be changed to something like this, since otherwise it seems to imply that the source index is altered: "The analytics job stores the annotated data, as well as a copy of the source data, in a new index."